packages/overture-schema/README.md

```diff
@@ -12,7 +12,32 @@ pip install overture-schema
 
 ## Usage
 
-This package serves as a test harness and validation framework.
+This package serves as a test harness and validation framework. Import specific schemas from their
+respective theme packages:
```

Comment:
This comment seems to be aimed at the same concern I was raising above about potential over-aggregation.

To the extent that it's a test harness, some alternatives:

1. Rename it to `overture-schema-test` or something like that.
2. (My more preferred). Do the first thing but also don't publish to PyPI.

Reply:
> The eventual version of this README shows that broader scope of this package. 

I'm still very skeptical of the package and its scope. This and the validation package are my two biggest discussion points/issues.